code/modules/economy/supply_misc.dm

```diff
@@ -116,6 +172,44 @@ ABSTRACT_TYPE(/area/supply)
 			if (prob(5))
 				qdel(src)
 
+/obj/strip_door/attackby(obj/item/I, mob/user)
+	if (ispryingtool(I))
+		playsound(src.loc, 'sound/items/Crowbar.ogg', 30, 1, -2)
+		src.change_direction()
+	if (isscrewingtool(I))
+		playsound(src.loc, 'sound/items/Screwdriver.ogg', 30, 1, -2)
+		SETUP_GENERIC_ACTIONBAR(user, src, 1 SECOND, /obj/strip_door/proc/screw_flaps, null, I.icon, I.icon_state, null, INTERRUPT_ACT | INTERRUPT_STUNNED | INTERRUPT_ACTION | INTERRUPT_MOVE)
+	if(src.flap_material)
+		if (issnippingtool(I))
+			SETUP_GENERIC_ACTIONBAR(user, src, 1 SECOND, /obj/strip_door/proc/snip_flaps, user, I.icon, I.icon_state, null, INTERRUPT_ACT | INTERRUPT_STUNNED | INTERRUPT_ACTION | INTERRUPT_MOVE)
+	else
+		if (istype(I, /obj/item/material_piece))
+			SETUP_GENERIC_ACTIONBAR(user, src, 1 SECOND, /obj/strip_door/proc/insert_flaps, I, I.icon, I.icon_state, null, INTERRUPT_ACT | INTERRUPT_STUNNED | INTERRUPT_ACTION | INTERRUPT_MOVE)
+	..()
```

Comment:
Specifically left it open, because I implemented it to work with any kind of material. Since in some industrial cases, things other than plastic would realistically see usage. I could limit it, but feel like it's just more fun to let people apply anything.

Reply:
Still seems very weird to just slap it with a cube of steel/etc. and have that just... work

Reply:
Suppose you're right, I just find the materials able to be turned into sheets a shame, guess people will just have to use MatSci to get sheets of items. Plastic might be an exception, or get actual sheets, I'll work something out.